extensions/helix/package.build.ts

```diff
@@ -137,6 +143,45 @@ export const pkg = (modules: Builder.ParsedModule[]) => ({
           }))(),
         },
 
+      surround: {
+        title: "Surround object with...",
+        items: {
+          "()": { command: "dance.run", args: { commands: [{ command: "dance.edit.insert", args: { shift: "extend", text: "(", where: "start" } }, { command: "dance.edit.insert", args: { shift: "extend", text: ")", where: "end" } }] }, text: "parenthesis block" },
+          "{}": { command: "dance.run", args: { commands: [{ command: "dance.edit.insert", args: { shift: "extend", text: "{", where: "start" } }, { command: "dance.edit.insert", args: { shift: "extend", text: "}", where: "end" } }] }, text: "braces block" },
+          "[]": { command: "dance.run", args: { commands: [{ command: "dance.edit.insert", args: { shift: "extend", text: "[", where: "start" } }, { command: "dance.edit.insert", args: { shift: "extend", text: "]", where: "end" } }] }, text: "brackets block" }, "<>": { command: "dance.run", args: { commands: [{ command: "dance.edit.insert", args: { shift: "extend", text: "<", where: "start" } }, { command: "dance.edit.insert", args: { shift: "extend", text: ">", where: "end" } }] }, text: "angle block" },
+          '"': { command: "dance.run", args: { commands: [{ command: "dance.edit.insert", args: { shift: "extend", text: '"', where: "start" } }, { command: "dance.edit.insert", args: { shift: "extend", text: '"', where: "end" } }] }, text: "double quote string" },
+          "'": { command: "dance.run", args: { commands: [{ command: "dance.edit.insert", args: { shift: "extend", text: "'", where: "start" } }, { command: "dance.edit.insert", args: { shift: "extend", text: "'", where: "end" } }] }, text: "single quote string" },
+          "`": { command: "dance.run", args: { commands: [{ command: "dance.edit.insert", args: { shift: "extend", text: "`", where: "start" } }, { command: "dance.edit.insert", args: { shift: "extend", text: "`", where: "end" } }] }, text: "grave quote string" },
```

Comment:
Could you use `Object.fromEntries(...)` or make a helper function here? The logic is largely the same, but it's quite hard to read.